helm-charts/FATE/values-template-example.yaml

```diff
@@ -28,17 +28,20 @@ modules:
   # - nginx
   # - rabbitmq
 
-# Computing : Eggroll, Spark, Spark_local
+# Computing : [Eggroll, Spark, Spark_local]
 computing: Eggroll
-# Federation: Eggroll(computing: Eggroll), Pulsar/RabbitMQ(computing: Spark/Spark_local)
+# Federation: [Eggroll(computing: Eggroll), Pulsar/RabbitMQ(computing: Spark/Spark_local)]
 federation: Eggroll
-# Storage: Eggroll(computing: Eggroll), HDFS(computing: Spark), LocalFS(computing: Spark_local)
+# Storage: [Eggroll(computing: Eggroll), HDFS(computing: Spark), LocalFS(computing: Spark_local)]
 storage: Eggroll
-# Algorithm: Basic, NN
+# Algorithm: [Basic, NN]
 algorithm: Basic
-# Device: IPCL, CPU
+# Device: [IPCL, CPU]
 device: IPCL
 
+skippedKeys:
+  - route_table
+
```

Comment:
Can we skip this key by default in golang code?

Reply:
We do hardcoded it into golang code，but there must be an example(could be another key). If not, the types of skippedKeys between the template config and the test config would be different so that the yaml validator would warn.